servers/zms/src/main/java/com/yahoo/athenz/zms/notification/RoleMemberExpiryNotificationTask.java

```diff
@@ -16,6 +16,8 @@
 
 package com.yahoo.athenz.zms.notification;
 
+import com.yahoo.athenz.auth.util.StringUtils;
```

Comment:
Why did we import this? We already have StringUtils in jetty so we should just use that class.